buildpacks/buildpack-clojure/buildpack-url

```diff
@@ -1 +1 @@
-https://github.com/heroku/heroku-buildpack-clojure
+https://github.com/heroku/heroku-buildpack-clojure
```

Comment:
Missing new line at eof. This might be related to the build issues.

Reply:
Weird that we'd need a newline.

Reply:
Newline was there before my edits, removed it to see if it was the issue, now added it agian, but the same end result remains....

```
#!/bin/bash -eo pipefail
tests=$(circleci tests glob buildpacks/*/tests/*/test.sh | circleci tests split --split-by=timings | xargs)
echo "executing tests: $tests"
basht $tests
Requested historical based timing, but they are not present.  Falling back to name based sorting
executing tests: buildpacks/buildpack-clojure/tests/clojure-ring/test.sh buildpacks/buildpack-java/tests/java-jetty/test.sh buildpacks/buildpack-play/tests/play/test.sh buildpacks/buildpack-scala/tests/scala/test.sh
=== RUN T_clojure-ring
::: BUILDING APP :::
       -----> Unable to select a buildpack
--- FAIL T_clojure-ring (1s)
=== RUN T_test_clojure-ring
-----> Unable to select a buildpack
--- FAIL T_test_clojure-ring (0s)
=== RUN T_java-jetty
::: BUILDING APP :::
       -----> Unable to select a buildpack
--- FAIL T_java-jetty (1s)
=== RUN T_play
::: BUILDING APP :::
       -----> Unable to select a buildpack
--- FAIL T_play (0s)
=== RUN T_scala
::: BUILDING APP :::
       -----> Unable to select a buildpack
--- FAIL T_scala (1s)

Ran 5 tests. 5 failed.

FAIL
Exited with code 5
```

Reply:
CircleCI passed before because `Dockerfile.dev` wasn't edited.

Reply:
@josegonzalez we didn't but it's good practice and it was there before this PR